src/foam/u2/TableView.js

```diff
@@ -158,9 +223,34 @@ foam.CLASS({
       name: 'onDAOUpdate',
       isFramed: true,
       code: function() {
-        this.data.select(foam.u2.TableBodySink.create({ properties_: this.properties_ }, this)).then(
-          function(a) { this.body = a.body; }.bind(this));
+        var dao = this.data;
+        if ( this.sortOrder ) {
```

Comment:
Ignore this for now.  If my DAOSlot work pans out we'll be replacing this anyways.